languages/rust/exercises/concept/tuples/.meta/example.rs

```diff
@@ -0,0 +1,21 @@
+pub enum Unit {
+    Millilitres,
+    UsCup,
+    UsFluidOnce,
+    UsTeaspoon,
+    UsTablespoon,
+}   
+
+pub fn get_volume(volume: &(Unit, f32)) -> f32 {
+    volume.1
+}
+
+pub fn to_cup(volume: &(Unit, f32)) -> (Unit, f32) {
+    unimplemented!("Solve me!")
```

Comment:
Could you perhaps implement all the methods in the example? That way we get to see best what is expected of the student.

Reply:
I have implemented some of the examples but not all possible combinations. I will do that once the Rust specific discussion occurs and is settled.